addons/common/functions/fnc_getArtilleryETA.sqf

```diff
@@ -0,0 +1,32 @@
+#include "script_component.hpp"
+/*
+ * Author: Kex
+ * Returns ETA to the target in seconds for given artillery unit based on
+ * target position and used magazine, -1 if target can't be hit.
+ * Also supports VLS.
+ *
+ * Arguments:
+ * 0: Vehicle <OBJECT>
+ * 1: Target position <ARRAY>
+ * 2: Ammo class <STRING>
+ *
+ * Return Value:
+ * ETA in seconds <NUMBER>
+ *
+ * Example:
+ * [artilleryUnit, getPos player, ammoClass] call zen_common_fnc_getArtilleryETA
+ *
+ * Public: No
+ */
+
+params [["_vehicle", objNull, [objNull]], ["_targetPosition", [0, 0, 0], [[]], 3], ["_ammoClass", "", [""]]];
+
+if (_vehicle call FUNC(isVLS)) then {
```

Comment:
If is VLS then exitWith? (perhaps)

Not overly sure about using exitWith for styling reasons but open for discussion.

Reply:
Negligible difference that is not worth discussing.